editor/src/components/inspector/sections/layout-section/conditional-section.tsx

```diff
@@ -76,27 +76,88 @@ const conditionSelector = createCachedSelector(
   },
 )((_, paths) => paths.map(EP.toString).join(','))
 
+const conditionExpressionSelector = createCachedSelector(
+  (store: MetadataSubstate) => store.editor.jsxMetadata,
+  (_store: MetadataSubstate, paths: ElementPath[]) => paths,
+  (jsxMetadata: ElementInstanceMetadataMap, paths: ElementPath[]): ConditionExpression => {
+    const elements = mapDropNulls((path) => {
+      const elementMetadata = MetadataUtils.findElementByElementPath(jsxMetadata, path)
+      if (
+        elementMetadata == null ||
+        !isRight(elementMetadata.element) ||
```

Comment:
just for consistency:
```suggestion
        isLeft(elementMetadata.element) ||
```